cob_obstacle_distance/scripts/obstacle_publisher_node.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2014 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_obstacle_distance
+ *
+ * \author
+ *   Author: Marco Bezzon, email: Marco.Bezzon@ipa.fraunhofer.de
+ *
+ * \date Date of creation: July, 2015
+ *
+ * \brief
+ *   Simple Python node to publish obstacles to cob_obstacle_distance node.
+ *
+"""
+import time
+import roslib 
+import rospy
+
+from geometry_msgs.msg import Pose
+from moveit_msgs.msg import CollisionObject
+from shape_msgs.msg import SolidPrimitive
+from shape_msgs.msg import Mesh
+
+
+if __name__=="__main__":
+    rospy.init_node("simple_obstacle_pub")
+    root_frame = "/odom_combined"
+        
+    pub = rospy.Publisher("/arm_right/obstacle_distance/registerObstacle", CollisionObject, queue_size=1)
```

Comment:
@ipa-fxm-mb 
have a look at the new [example_obstacle_publisher_node script](https://github.com/ipa-fxm/cob_control/blob/mesh_repr_of_registered_links/cob_obstacle_distance/scripts/example_obstacle_publisher_node.py)...

using collada instead of stl works well!